File: 프로그래머스/0/120808. 분수의 덧셈/분수의 덧셈.js
function solution(numer1, denom1, numer2, denom2) {
    let num1 = (numer1 * denom2) + (numer2 * denom1), num2 = denom1 * denom2, max = 1;

    for (let i = 1; i <= num1; i++) {
        if (num1 % i == 0 && num2 % i == 0) {
            max = i;
        }
    }
    return [num1 / max, num2 / max];
}

File: 프로그래머스/0/120808. 분수의 덧셈/분수의 덧셈.py
import math

def solution(numer1, denom1, numer2, denom2):
    new_numer = (numer1 * denom2) + (numer2 * denom1)
    new_denom = denom1 * denom2
    common_divisor = math.gcd(new_numer, new_denom)
    return [new_numer // common_divisor, new_denom // common_divisor]
